### DISCUSSION TITLE
Update Error v0.2.3-hf2

### DISCUSSION THREAD

**Discussion by Sedamix141** [maxrave-dev/SimpMusic#555] [Q&A]
[Screenshot_20241107_172418_SimpMusic.jpg] (https://github.com/user-attachments/assets/fe619637-78ff-4ac4-9217-43de3132f782) 

Download the latest version but keep asking me to update delete the apps and reinstall it but still have the same flaw that can be done in that case..?

**Comment by ThinethSankalpa** [comment]
Now it's fixed. Thank you very much developer. I'll buy you a coffee if I had money 🫤

**Comment by ilianoKokoro** [comment]
@Sedamix141 The issue is now fixed. Make sure to update to the latest version.
